### PR TITLE
Node.js bindings: Add a sample for SPI JS bindings

### PR DESCRIPTION
**Pull request by nagineni**
This adds a sample code to demonstrate the usage of SPI JS bindings for configuring and reading the temperature from the MAX31855 thermocouple amplifier connected to the MinnowBoard MAX/Turbot.

Replaces #1959 

Signed-off-by: Sudarsana Nagineni <sudarsana.nagineni@intel.com>

### CODE REVIEW

**Comment by nagineni** [comment]
@edersondisouza I fixed the review comment. PTAL.

Thanks!

File: src/samples/nodejs/spi_max31855.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    spiBus = null;
+
+spi.open( {
+    bus: 0,
+    frequency: 2000000
+} ).then( function( bus ) {
+    spiBus = bus;
+
+    // Take a temperature reading in every one second
+    readInterval = setInterval( function() {
```
Comment (edersondisouza) [inline src/samples/nodejs/spi_max31855.js]: Shouldn't you have declared this before, like `spi` and `spiBus`? As you use it on signal handler?

Comment (edersondisouza) [inline src/samples/nodejs/spi_max31855.js]: If so, you may just fix it and merge =D

Comment (nagineni) [inline src/samples/nodejs/spi_max31855.js]: Ah! I missed it again :(. Thanks for the green light. I will fix and merge this.

**Pull request closed without merging**